app/assets/javascripts/autocomplete.js.coffee

```diff
@@ -36,7 +36,7 @@
     limit: 10
   ).on("typeahead:selected typeahead:autocompleted", (event, suggestion, dataset) ->
     target = $("##{$t.data('target')}")
-    target.val suggestion["id"]
+    target.val suggestion["id"] || suggestion["display"]
```

Comment:
I think it is sensible to have this as a fallback, but I think it would also be good to have `MediaObject#autocomplete` return `id` values as well so it matches the interface for autocomplete methods.  (We need to make this interface explicit somewhere in the code, probably on `ObjectsController#autocomplete`, but that is not in scope of this PR.)  I'm thinking `id` and `display` values would be the same for the series data.